cms/articles/models.py

```diff
@@ -446,9 +445,39 @@ def figures_used_by_ancestor(self) -> list[str]:
         topic_page_class = resolve_model_string("topics.TopicPage")
         topic = topic_page_class.objects.ancestor_of(self).first().specific_deferred
         return [
-            figure.value["figure_id"] for figure in topic.headline_figures if figure.value["series"].id == series.id
+            figure.value["figure_id"]
+            for figure in topic.headline_figures
+            if figure.value["series"].id == series.id
+            and figure.value["figure_id"] in self.headline_figures_figure_ids_list
         ]
 
+    @property
+    def figures_in_use_with_no_fallback(self) -> set[str]:
+        """Returns the set of figure IDs used by the ancestor topic page, for which there are no values to fall back on
+        from the previous to latest article in the series.
+        """
+        if not self.is_latest:
+            # Figures are only ever used from the latest article in the series
+            return set()
+
+        figures_in_use = set(self.figures_used_by_ancestor)
+        if not figures_in_use:
+            return figures_in_use
+
+        articles_headline_figures = (
+            StatisticalArticlePage.objects.sibling_of(self)
+            .live()
```

Comment:
Let's still check for `.public()`

Reply:
I'm not sure if that check is appropriate here, I believe the topic page would happily pull figures from an article even if had view restrictions. We currently allow a private article to be considered latest.